js/start.js

```diff
@@ -584,14 +582,18 @@ function start() {
 
           localStorage.serverIdAtLastStart = curConn && curConn.server && curConn.server.id;
 
-          if (app.localSettings.get('shareMetrics') === undefined) {
+          const metricsOn = app.localSettings.get('shareMetrics');
+          const metricsNewV = app.localSettings.get('mVersion') !== mVersion();
```

Comment:
This is optional, but would it be better to check if the version is smaller rather than not equal? This wouldn't come into play in prod, but would for the devs going between branches.

(there's another place this check it made as well)